src/shared/use-server-time-as-local/use-server-date-time-as-local.js

```diff
@@ -0,0 +1,21 @@
+import { useConfig } from '@dhis2/app-runtime'
+
+export const useServerDateTimeAsLocal = (dateTime) => {
+    const { systemInfo } = useConfig()
+
+    if (!dateTime) {
+        return undefined
+    }
+
+    const localNow = new Date()
+    localNow.setMilliseconds(0)
+    const nowAtServerTimeZone = new Date(
```

Comment:
This has some history to it. This was added to resolve a bug. But that was in another app, where we were always passing date/time strings comping from the server. Now these strings are only accurate to the second, and what was happening, especially on a local instance, was the following:
- We were doing a mutation and when the mutation was completed we would fetch the same data from the server.
- Sometimes this would happen in under a second
- And because the local now was accurate to the millisecond but the server date string was not (I guess they were also rounding things server side), we could get mismatches.
- This resulted in us sometimes displaying weird "time ago" messages, such as "This was approved a few seconds from now"

But, I will still keep this in mind when I rework this PR, since you suggested we should be able to pass in a date instance, and also it would just look more consistent as you suggested already.

Reply:
This whole spiel turned out to no longer be relevant, because I removed this hook.